webhook_handlers/views/github.py

```diff
@@ -265,9 +267,11 @@ def push(self, request, *args, **kwargs):
                     commits=[commit.get("id") for commit in commits],
                 ),
             )
+        else:
```

Comment:
Had to do it this way because after a django update, if your original query is no longer satisfied, which was the case when we updated the branch to pushed_to_branch_name, then your queryset no longer uses records. This is the same logic because both if's do an update, only difference is that the first if also does the merged=True